project/MimaExcludes.scala

```diff
@@ -73,7 +81,11 @@ object MimaExcludes {
           "org.apache.spark.ml.regression.LeastSquaresCostFun.this"),
         ProblemFilters.exclude[MissingMethodProblem](
           "org.apache.spark.ml.classification.LogisticCostFun.this"),
-        // SQL execution is considered private.
+        ProblemFilters.exclude[MissingMethodProblem](
```

Comment:
Are they necessary for 1.5? It would be nice to group rules from `LogisticAggregator`.